src/screens/UnknownTokens.js

```diff
@@ -63,7 +63,7 @@ class UnknownTokens extends React.Component {
     for (const token of this.props.allTokens) {
       // If has balance but does not have token saved yet
       if (this.props.registeredTokens.find((x) => x.uid === token) === undefined) {
-        const filteredHistoryTransactions = hathorLib.wallet.filterHistoryTransactions(this.props.historyTransactions, token);
+        const filteredHistoryTransactions = hathorLib.wallet.filterHistoryTransactions(this.props.historyTransactions, token, false);
```

Comment:
The argument list is becoming too big. Maybe we can refactor to send an object instead of individual arguments. Something like:

```
filterHistoryTransactions = ({ arg1, arg2, arg3}) => {
}
```

Then, we would be able to call this method using any order, and we would be forced to name each argument. What do you think?

Reply:
I don't really like sending an object for all arguments, it opens the method for a lot of bad uses, I think. It's not difficult to change, but I don't think it's good. Why do you think 4 arguments is too much?

Reply:
You're right. I don't think it it too much. But it is not intuitive what the last argument `false` means. I also agree that it is too flexible, and we can skip my suggestion.